packages/common/components/blocks/external-api.marko

```diff
@@ -1,7 +1,7 @@
 import fetch from "node-fetch";
 
 $ const { log } = console;
-$ const { apiUrl, ctaLinkStyle } = input;
+$ const { apiUrl, shortName, ctaLinkStyle } = input;
```

Comment:
I don't believe this is used in this file or `common-content-api-list-item-block` via it's `input` unless I'm misunderstanding something here?